### ISSUE TITLE
Proposed File Format + file type

### ISSUE THREAD

**Issue by superslalomracer** [kevinaangstadt/CS340-Summer21-Notes#11]
I'm currently working on my notes for Day 4, and I think we should agree on a general file type and format. 

Since we're supposed to write in markdown, I was thinking HTML for ease of editing and because it can be opened anywhere. I was also thinking we should make a file with a general format for all the lecture notes so everyone can pull it and just conform to the format. If that sounds good, I can make a preliminary file, then whoever wants to can edit it until like a week later, then we'll all conform to that version.

**Comment by DigitalSRC** [comment]
If you ain't gonna suggest one I got one. I suggest we not use HTML that complicates things, id rather not format in HTML nor do I want to learn it at the moment. I think it's easiest to use ".md" files (markdown files). As for my proposal for file formating, I've broken my file into 3 categories: Commands, Programs, Notes.

The first section is all about commands used/learned during class. This is so we have an easy reference to what we used that day. The program category is the same but it's just to denote that these are other programs, so they have subcommand. Both these two categories use bullet points. The final is the Note section which is formated in a more readable way but can include bullet points for areas that need them etc. but its more text.
